blocks/related-resources/related-resources.js

```diff
@@ -33,10 +32,10 @@ async function setRowDetails(row, block) {
           break;
         case 'H4':
         case 'H5':
-          row.assetType = el.innerHTML;
+          row.assettype = el.innerHTML;
           break;
         case 'P':
-          row.description = `<p>${el.innerHTML}</p>${row.description}`;
+          row.description = `<p>${el.innerHTML}</p>`;
```

Comment:
what if there are multiple paragraphs of text? then you're only picking the last <p> tag essentially. You might want to concatenate the <p> tags into the description.

Reply:
Here's an example of multiple paragraphs in a card, FYI https://www.merative.com/support/imaging-product-support-details/tuesdays-with-merative-imaging